services/src/test/java/io/scalecube/services/GreetingServiceImpl.java

```diff
@@ -9,6 +9,11 @@
 final class GreetingServiceImpl implements GreetingService {
 
   @Override
+  public String toString() {
+    return "GreetingServiceImpl []";
```

Comment:
are you testing it somehow via toString? in such case maybe better to test in some more straight forward way.